enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -373,7 +464,7 @@ items:
     ownerReferences:
     - apiVersion: v1
       kind: Service
-      name: istio-ingressgateway
+      name: example-gateway-loadbalancer-service
```

Comment:
Shouldn't this just be:
```suggestion
      name: example-gateway
```

Going with the example above, the names don't match.

Reply:
Fixed in https://github.com/openshift/enhancements/compare/732fe4ef4bfdf98263827650e5ca5127479bc8d1..5ccb87fd14dd84eebfc38506dc1596a75eeccdfb#diff-7973eca3c2f112ece8f8585f4f4c6567fc1c1b4fa95be7f88a27ce08820d89f7.